catch/unit/callback/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2022 Advanced Micro Devices, Inc. All Rights Reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+# Common Tests - Test independent of all platforms
+
+set(TEST_SRC
+    hipApiName.cc
+    hipGetStreamDeviceId.cc
+    hipKernelNameRef.cc
+    hipKernelNameRefByPtr.cc
+)
+
+add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/SimpleKernel.code
+                   COMMAND ${CMAKE_CXX_COMPILER} --genco --std=c++17 ${CMAKE_CURRENT_SOURCE_DIR}/SimpleKernel.cc -o SimpleKernel.code
+                   DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/SimpleKernel.cc)
```

Comment:
genco does not work properly on windows yet. you will need to exclude this test building on windows. additionally you are missing specifying the offload arch & output path for building the code object. see unit/deviceLib/CMakeLists.txt for an example of how to do it for all of the above.

Reply:
@mangupta: Review comment addressed within the commit https://github.com/ROCm-Developer-Tools/hip-tests/pull/6/commits/8cf9d51dd67d0f050dfd1914ee52412f15df50ed. Can you please check if this patch has solved a mentioned issue on Windows?